modin/pandas/series.py

```diff
@@ -1689,6 +1689,9 @@ def _validate_dtypes_min_max(self, axis, numeric_only):
     def _validate_dtypes(self, numeric_only=False):
         pass
 
+    def _numeric_data(self, axis):
+        return self
+
```

Comment:
Add comments that `numeric_only` parameter is not implemented for series, so this function doesn't really do anything.

Reply:
done